tests/all/integration/api/helpers/test_order.py

```diff
@@ -54,6 +56,48 @@ def test_should_delete_related_attendees(self):
             order = db.session.query(Order).filter(Order.id == obj.id).first()
             self.assertEqual(len(order.ticket_holders), 0)
 
+    def test_count_sold_tickets(self):
+        """Method to test the count query of sold tickets"""
+
+        with app.test_request_context():
+            ticket = TicketFactory()
+
+            completed_order = OrderFactory()
+            completed_order.status = 'completed'
+
+            placed_order = OrderFactory()
+            placed_order.status = 'placed'
+
+            initialized_order = OrderFactory()
+            initialized_order.created_at = datetime.utcnow() - timedelta(minutes=5)
+
+            expired_time_order = OrderFactory()
+            expired_time_order.created_at = common.date_
+
+            expired_order = OrderFactory()
+            expired_order.status = 'expired'
+
+            db.session.commit()
+
+            first_att_with_no_order_id = AttendeeFactoryBase()
+            second_att_with_no_order_id = AttendeeFactoryBase()
```

Comment:
Can be done in a single line, and no need to assign when you're not going to use